tcms/core/tests/test_admin.py

```diff
@@ -44,21 +43,22 @@ def test_admin_display(self):
 
         self.assertNotContains(response, 'Test case status')
         self.assertNotContains(response, 'Test case texts')
-        self.assertNotContains(response, 'Test cases')
+        self.assertContains(response, 'Search Test Cases')
 
         # for tcms.testplans
         self.assertContains(response, 'Plan types')
 
-        self.assertNotContains(response, 'Test plans')
+        self.assertContains(response, 'Search Test Plans')
```

Comment:
Similar story with "Test plans", the assert matches the navigation menu item, not the admin page. ATM there is a model admin for test plans.